src/main/resources/discord-config-prod.yml

```
@@ -1,4 +1,5 @@
 discord:
   webhook-url:
     attendance: ${DISCORD_ATTENDANCE_HOOK}
-    activity: ${DISCORD_ACTIVITY_HOOK}
\ No newline at end of file
+    activity: ${DISCORD_ACTIVITY_HOOK}
+    statistics: ${DISCORD_STATISTICS_HOOK}
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)